Add command "nomad tls" (#14296)
hashicorp/nomad: website/content/docs/commands/tls/cert.mdx

---
layout: docs
page_title: 'Commands: TLS Cert Create'
---

> # Nomad TLS Cert Create
> 
> Command: `nomad tls cert create`

Reviewer: There's a format that all the command docs share, so copy those:

```suggestion
# Command: nomad tls cert create
```

(Applies to all the docs)

> The `tls cert create` command is used to create certificates to be used in 
> for your Nomad cluster for TLS setup. You can then copy these to your server
> as part fo the configurstion. This command will not automatically update the 
> settings in the agents.

Reviewer: Reworded for clarity/spelling. Also, because this is a web page and not the CLI help text, we can link out to the guide:

```suggestion
The `tls cert create` command is used to create certificates to be used for
[TLS encryption][] for your Nomad cluster. You can then copy these to your 
servers and clients. This command will not automatically update the 
configuration of the agents.
```

And then add a footnote at the bottom to make it a clickable link:

```
[TLS encryption]: https://learn.hashicorp.com/tutorials/nomad/security-enable-tls?in=nomad/transport-security
```
